components/OAuth2.0/OauthHttpLocalServer.c

```diff
@@ -188,3 +187,29 @@ bool Spotify_StartMDNSService()
     ESP_LOGI(TAG, "MDNS Init done");
     return true;
 }
+
+/**
+ * @brief Run Http local service
+ */
+bool HttpServerServiceInit()
+{
+    SendCodeFromHttpToTask = 
+            xQueueCreate(1, sizeof(char) * sizeof(char[MEDIUM_BUF]));
+    
+    httpd_handle_t serviceLocalServer = Oauth_StartWebServer();
+    if (serviceLocalServer == NULL)
+    {
+        ESP_LOGE(TAG, "Creating Service local server failed!");
+        return false;
+    }
+    
+    bool IsMdnsStarted = Oauth_StartMDNSService();
+    if (!IsMdnsStarted)
+    {
+        ESP_LOGE(TAG, "Running mDNS failed!");
+        return false;
+    };
+
+    ESP_LOGI(TAG, "** local server created, mDNS is running! **");
```

Comment:
it is a good idea to print the mdns `hostname` in the log!

Reply:
done.